plans/chew-large-datasets/README.md

```diff
@@ -54,7 +54,6 @@ IPFS supports an ever-growing set of ways in how a File or Files can be added to
 ### `Test:`  IPFS Add Dir Sharding
 
 - **Test parameters**
-  - `File Sizes` - An array of File Sizes to be tested (default to: `[1MB, 1GB, 10GB]`)
```

Comment:
No more File Sizes?

Reply:
Note that File Sizes are useful for the tests that only add files and not directories

Reply:
This test in specific does not require file sizes as it is directory-only.

Reply:
But I'll add more test sizes!

Reply:
Ah! you are right! I misread :)